### PR TITLE
Logo template

### PR DESCRIPTION
**Pull request by priyanka-surana**
This merge request adds the pipeline logo template `pptx` file and a few steps to use the template.

### CODE REVIEW

File: docs/images/how_to_use_logo_template.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# How to use the logo template
+
+1. Replace `pipelinename` with the actual name of the pipeline
+2. Save the slide as a `sanger-tol-<pipeline>_logo.png`
```
Comment (muffato) [inline docs/images/how_to_use_logo_template.md]:
```suggestion
2. _Export_ the slide as a `sanger-tol-<pipeline>_logo.png`
```

**Review by muffato — CHANGES_REQUESTED** [review]
Can you add the recommended dimensions of the image ?

File: docs/images/how_to_use_logo_template.md (hunk 1 above)
Comment (priyanka-surana) [reply docs/images/how_to_use_logo_template.md]: Added the minimum resolution

Comment (muffato) [reply docs/images/how_to_use_logo_template.md]: How do you export the slide ? When I click "Export: and select PNG, I'm asked for a resolution in pixels, not pixels/inch
Screenshot 2022-07-20 at 11 54 35

Comment (priyanka-surana) [reply docs/images/how_to_use_logo_template.md]: I have made the changes to include those descriptors as well. I generally do a default export and then use tools in preview to make sure I have the right resolution.

**Review by muffato — APPROVED** [review]

File: docs/images/how_to_use_logo_template.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# How to use the logo template
+
+1. Replace `pipelinename` with the actual name of the pipeline.
+2. Export the slide as `sanger-tol-<pipeline>_logo.png` with a width of 2999 pixels, a height of 1080 pixels, and target a resolution of 300 pixels per inch.
```
Comment (muffato) [inline docs/images/how_to_use_logo_template.md]:
```suggestion
2. Export the slide as `sanger-tol-<pipeline>_logo.png` with a width of 3000 pixels, a height of 1080 pixels, and target a resolution of 300 pixels per inch.
```

**Pull request merged**